src/OrchardCore/OrchardCore/Modules/ModularBackgroundService.cs

```diff
@@ -187,7 +200,7 @@ private async Task UpdateAsync(IEnumerable<ShellContext> previousShells, IEnumer
                         }
                         catch (Exception ex) when (!ex.IsFatal())
                         {
-                            _logger.LogError(ex, "Error while getting the time zone of the tenant '{TenantName}'.", tenant);
+                            _logger.LogError(ex, "Error while getting the site settings of the tenant '{TenantName}'.", tenant);
```

Comment:
Hmm, isn't this still the time zone **and** site settings though?

Reply:
I do the same in another place to get the base url from the site settings, in both cases it may fail when getting the site settings, but the time zone / base url can be null without failing.

But you're right, better to be more specific, i will change it here by `Error while getting the time zone from the site settings`, and in the other place by `Error while getting the base url from the site settings`.